server/methods/accounts/accounts.js

```diff
@@ -910,15 +912,21 @@ export function setUserPermissions(userId, permissions, group) {
  * @return {String} Email logo path
  */
 function getEmailLogo(shop) {
-  let emailLogo;
   if (Array.isArray(shop.brandAssets)) {
-    const brandAsset = _.find(shop.brandAssets, (asset) => asset.type === "navbarBrandImage");
-    const mediaId = Media.findOne(brandAsset.mediaId);
-    emailLogo = path.join(Meteor.absoluteUrl(), mediaId.url());
-  } else {
-    emailLogo = Meteor.absoluteUrl() + "resources/email-templates/shop-logo.png";
+    let media;
```

Comment:
the changes from here add extra null-checking.
I changed them to make tests a little easier to write (which is typically not a great reason for changing code, I know!), but also figured extra null checks are never a bad thing.